### PR TITLE
Reword `curl | sh` disclaimer in TRPL §2.1

### PR DESCRIPTION
**Pull request by kini**
The paragraph here seemed confusing, so I reworded it.  Also added
another possible reason why `curl | sh` might be objectionable to users.

r? @steveklabnik

### CODE REVIEW

File: src/doc/trpl/installing-rust.md [hunk 1]
```diff
@@ -51,11 +51,15 @@ us an uninstall option.
...
 
 Some people, and somewhat rightfully so, get very upset when we tell them to
-`curl | sh`. Basically, when they do this, they are trusting that the good
-people who maintain Rust aren't going to hack their computer and do bad things.
-That's a good instinct! If you're one of those people, please check out the
-documentation on [building Rust from Source][from-source], or [the official
-binary downloads][install-page].
+`curl | sh`. Their concern is that `curl | sh` implicitly requires you to trust
+that the good people who maintain Rust aren't going to hack your computer and
+do bad things -- and even having accepted that, there is still the possibility
```
Comment (steveklabnik) [inline src/doc/trpl/installing-rust.md]: can this be an em dash instead of two hyphens please?

Comment (kini) [inline src/doc/trpl/installing-rust.md]: Sure. Do you want that as an HTML entity or a literal unicode character?

Comment (steveklabnik) [inline src/doc/trpl/installing-rust.md]: Either is fine, I guess literal unicode, so it looks nice elsewhere

Comment (kini) [inline src/doc/trpl/installing-rust.md]: Fixed in 968d17a.

**Comment by steveklabnik** [comment]
Sorry about the delay @kini ! Github doesn't give a notification when you push new commits, so I didn't see till now :/

**Comment by kini** [comment]
Ah, no problem. That's good to know. From now on I'll leave a comment after pushing new commits. I actually did leave a comment on the thread we had going on the outdated diff, but maybe that didn't send a notification because the diff was outdated.

**Comment by steveklabnik** [comment]
Ah! I think I probably just missed that one, then :(

**Comment by kini** [comment]
Don't worry about it :) Thanks for reviewing! I assume @bors will close this PR automatically once it's merged?

**Comment by steveklabnik** [comment]
yup!

**Pull request merged**